pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -854,8 +861,9 @@ public static void deactivateKitRequest(int dsmKitRequestId, @NonNull String dea
             kitRequestShipping.setDsmKitRequestId(dsmKitRequestId);
             kitRequestShipping.setDeactivationReason(deactivationReason);
             kitRequestShipping.setDeactivatedDate(deactivatedDate);
-
+            kitRequestShipping.setKitTypeName(kitRequestDao.getKitTypeByKitRequestId(dsmKitRequestId));
```

Comment:
Sorry, my question may have been unclear.
Is this still necessary now that you get the value via the select?

Reply:
Yes Dennis. This part of code is not using that SELECT statement , actually not doing a DB query, hence need to query and get it. 
line 860: KitRequestShipping kitRequestShipping = new KitRequestShipping();

Reply:
I am confused then.
What was the purpose of the changes to this file in the most recent commit?

Reply:
Its used in other place reactivateKitRequest method and avoided extra separate query to DB. 
I thought , thats what you recommended in previous comment

Reply:
No, I was commenting about `kitRequestDao.getKitTypeByKitRequestId`, since it is making a second query while processing the results of the first query.
I thought perhaps you could just add the fields you needed to the first query to avoid the second query.

I'm not sure I see the other place you mentioned, but will go back through the commits to try to figure that out.